examples/lead-agent/backend/app.py

```
@@ -0,0 +1,147 @@
+            candidate_id=data.get('candidate_id', 'temp-id'),
+            name=data['name'],
+            email=data.get('email', ''),
+            bio=data['bio'],
+            skills=data.get('skills', ''),
+            job_description=data.get('job_description', ''),
+            additional_instructions=data.get('additional_instructions', '')
+        )
+        
+        return jsonify(result), 200
+        
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)